docs/tendermint-core/validators.md

```diff
@@ -22,9 +22,72 @@ Validators have a cryptographic key-pair and an associated amount of
 
 There are two ways to become validator.
 
-1.  They can be pre-established in the [genesis state](./using-tendermint.md#genesis)
-2.  The ABCI app responds to the EndBlock message with changes to the
-    existing validator set.
+1. They can be pre-established in the [genesis state](./using-tendermint.md#genesis)
+2. The ABCI app responds to the EndBlock message with changes to the
+   existing validator set.
+
+## Setting up a Validator
+
+When setting up a validator there are countless ways to configure your setup. This guide is aimed at showing one of them, the sentry node design. This design is mainly for DDOS prevention.
+
+### Network Layout
+
+![ALT Network Layout](./sentry_layout.png)
+
+The diagram is based on AWS, other cloud providers will have similar solutions to design a solution. Running nodes is not limited to cloud providers, you can run nodes on bare metal systems as well. The architecture will be the same no matter which setup you decide to go with.
+
+The proposed network diagram is similar to the classical backend/frontend separation of services in a corporate environment. The “backend” in this case is the private network of the validator in the data center. The data center network might involve multiple subnets, firewalls and redundancy devices, which is not detailed on this diagram. The important point is that the data center allows direct connectivity to the chosen cloud environment. Amazon AWS has “Direct Connect”, while Google Cloud has “Partner Interconnect”. This is a dedicated connection to the cloud provider (usually directly to your virtual private cloud instance in one of the regions).
```

Comment:
Do we generally not recommend running validators in cloud environments (even private ones), only in private data centers?

Reply:
People have recommended not to do this, but quite a few do anyways. I tried to leave it generalized, but may need a bit more work.